src/scaffoldmaker/meshtypes/meshtype_3d_bladder1.py

```diff
@@ -346,25 +352,25 @@ def getDefaultOptions(cls, parameterSetName='Default'):
             'Refine number of elements through wall': 1
         }
         if 'Cat 1' in parameterSetName:
-            options['Wall thickness'] = 0.008  # was 1.5 / 178.08271473110773
+            options['Wall thickness'] = 0.007  # was 1.5 * 2.0 / 178.08271473110773
             options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Human 1' in parameterSetName:
-            options['Wall thickness'] = 0.009  # was 3.0 / 327.63
+            options['Wall thickness'] = 0.043  # was 3.0 * 2.0 / 327.63
             options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Mouse 1' in parameterSetName:
-            options['Wall thickness'] = 0.007  # was 0.5 / 72.004931756029
+            options['Wall thickness'] = 0.008  # was 0.5 * 2.0 / 72.004931756029
             options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Pig 1' in parameterSetName:
-            options['Wall thickness'] = 0.005  # was 2.5 / 505.2045520616655
+            options['Wall thickness'] = 0.016  # was 2.5 * 2.0 / 505.2045520616655
             options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Rat 1' in parameterSetName:
-            options['Wall thickness'] = 0.003  # was 0.3 / 110.40996641878101
+            options['Wall thickness'] = 0.006  # was 0.3 * 2.0 / 110.40996641878101
             options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Material' in parameterSetName:
             options['Number of elements along dome'] = 8
             options['Number of elements along neck'] = 4
             options['Number of elements around'] = 8
-            options['Wall thickness'] = 0.01
+            options['Wall thickness'] = 0.016  # an average of the wall thicknesses for above species
             options['Ureter position around'] = 0.67
```

Comment:
Since we require the 'Ureter position around' to be the same for all variants, remove it from the per-species and material code. Add a comment explaining that it's a fixed material coordinate and thus not to be edited.
Remove this option from the visible options in getOrderedOptionNames() below.

Reply:
Done as suggested above.